server/app/jobs/RecurringJobExecutionTimeResolver.java

```diff
@@ -0,0 +1,18 @@
+package jobs;
+
+import java.time.Clock;
+import java.time.Instant;
+
+/**
+ * Implementations of {@code RecurringJobExecutionTimeResolver} determine when a recurring {@link
+ * DurableJob} should next run.
+ */
+@FunctionalInterface
+public interface RecurringJobExecutionTimeResolver {
+
+  /**
+   * Takes an {@link Clock} with the local time zone and returns an instant representing when the
```

Comment:
Comment style is off

Should be more like "Returns the next execution time based on the local time settings represented by {@code clock}"

"Takes a clock" and "returns an instance" are clear parts of the method signature and get in the way of what the method does. Imp details about either can be in @param of @return if not obvious otherwise.

Reply:
Done